dragonfly/story.py

```diff
@@ -150,23 +172,37 @@ def floor_height(self):
 
     @property
     def volume(self):
```

Comment:
I think instead of making this always with multiplier we either should change them to methods with an input for multiplier or create a new property (e.g. volume_total or volume_multiplied). I think it will be more common that one only wants the volume or floor area of a single story. They can of course divide the number by self.multiplier but they should not have to.

Reply:
There was no ambiguity with the way I had it previously, where multipliers were on the `Building` level and it was clear that `volume` on the `Building` accounted for multipliers while it did not do so on the `Story` level. I think you are right that these properties like `volume` should go back to the way that they were and this is also in-line with how we implemented multipliers on Rooms in honeybee-core.  I think you're also right that the volume of just a single `Story` is likely to be more common. 

I also feel like the act of a simple multiplication isn't enough to justify a whole new property here. If people want the `volume_total`, they can just access the story volume and multiplier and figure it out from there. Let's keep it simple for now and only add a `volume_total` property if we find that we need it.